experimental/dds/tree2/src/feature-libraries/forestRepairDataStore.ts

```diff
@@ -46,8 +46,17 @@ export class ForestRepairDataStore implements RepairDataStore {
 
 	public capture(change: Delta.Root, revision: RevisionTag): void {
 		const forest = this.forestProvider(revision);
+		/**
+		 * Cursor used to traverse the forest and build fetch the repair data.
+		 * Note that the cursor is implicitly captured by the functions below, which have requirements for the cursor.
+		 * Calling those functions requires that the cursor be in the appropriate state.
+		 */
 		const cursor = forest.allocateCursor();
 
+		/**
+		 * Requires that `cursor` be positioned on a node.
```

Comment:
I think it would be better to say: 
```suggestion
		 * Visits the node `cursor` is positioned at.
```

since nothing currently says what node this visits.